### ISSUE TITLE
Reader Conversations: make comments with errors visible

### ISSUE THREAD

**Issue by bluefuton** [Automattic/wp-calypso#18701]
Currently, when posting a comment with an error in Conversations, the comment vanishes completely.

I think this is because the placeholder comment is not in the list of expanded comments. We could potentially add the placeholder comment to `state.comments.expansions` when encountering a `COMMENT_WRITE_ERROR`.

cc @samouri

**Comment by fraying** [comment]
Is this the same as #18595?

**Comment by bluefuton** [comment]
> Is this the same as #18595?

Yes! It's closely related. Whereas successful new comments vanish into the caterpillar and you can at least expand them, you can never get back to the text box to edit the comment if there's an error.

**Comment by fraying** [comment]
@bluefuton Aha! Cool - thanks for explaining.

**Comment by samouri** [comment]
> We could potentially add the placeholder comment to state.comments.expansions when encountering a COMMENT_WRITE_ERROR.

That makes sense to me @bluefuton! Will probably want to set it to fully expanded as well
